main.go

```diff
@@ -110,22 +111,38 @@ func main() {
 	flag.StringVar(&a.apiKey, "api-key", "", "Uptime Robot API key")
 	flag.StringVar(&a.address, "ip", "0.0.0.0", "IP on which the Prometheus server will be binded")
 	flag.StringVar(&a.port, "p", "9705", "Port that will be used by the Prometheus server")
+	flag.StringVar(&a.logLevel, "log-level", "info", "Log Level. Supported log levels are trace, debug, info, warn, error, fatal, panic")
 	flag.IntVar(&a.scrapeInterval, "inteval", 30, "Uptime robot API scrape interval, in seconds")
 	flag.Parse()
 
+	zerolog.TimeFieldFormat = zerolog.TimeFormatUnix
+
+	logLevel, err := zerolog.ParseLevel(a.logLevel)
+	if err != nil {
+		log.Error().
+			Err(err).
+			Msg("invalid log level defined. Setting default to info level.")
+		a.logLevel = "info"
+	}
+	zerolog.SetGlobalLevel(logLevel)
+
 	if a.apiKey == "" {
 		a.apiKey = os.Getenv("UPTIMEROBOT_API_KEY")
 		if a.apiKey == "" {
-			logrus.Fatal(errors.New("no API key provided in flags nor in env variables"))
+			log.Fatal().
```

Comment:
This will never be executed:

```
$ echo $UPTIMEROBOT_API_KEY

$ make build && ./out/bin/uptimerobot-exporter
mkdir -p out/bin
GO111MODULE=on go build -o out/bin/uptimerobot-exporter .
{"level":"info","time":1626279652,"message":"API key found"}
^C
```
As you can see, even with API key, the programs takes the happy path. You can read from the zerolog documentation:
>Fatal starts a new message with fatal level. The os.Exit(1) function is called by the Msg method, which terminates the program immediately.
>
>You must call Msg on the returned event in order to send the event.

So the `.Msg()` function must be called for each log using the fatal level. A generic message such as "missing API key" should do the work.

Reply:
Thanks, I completely missed that, just calling Fatal won't terminate.